certifier_service/certlib/certlib_support.go

```diff
@@ -64,9 +64,9 @@ func PrintDominanceNode (ind int, node *PredicateDominance) {
 }
 
 func PrintDominanceTree(ind int, tree *PredicateDominance) {
-	PrintDominanceNode (ind, tree)
+	PrintDominanceNode(ind, tree)
 	for n := tree.FirstChild; n != nil; n = n.Next {
-		PrintDominanceTree(ind + 2, n)
+		PrintDominanceTree(ind+2, n)
```

Comment:
Some of the default `gofmt` rules takes out spaces. I think the old code is more readable ... but we'll have to just live with this no-space formatting for now.

I did investigate ways to profile gofmt-formatting, but didn't find anything very customizable.

I think in the longer-term we will need a way to specify format-rules, to make the code a bit more readable per out local styles.